source/shared/net/concepts/concepts-delegation.rst

```diff
@@ -4,39 +4,43 @@
 
 Pools and Rewards
 =================
 
-A baker opens a pool, and other CCD holders delegate some stake to this pool. The probability that the baker running the pool is chosen to bake the next block is then proportional to the total stake in the pool. Rewards are distributed to everyone in the pool proportional to their stake, and a commission is paid to the baker by all delegators. For example, suppose that a pool has 10,000 CCD. And suppose that in a 24 hour period this pool gets 30 CCD as reward and the commission of the baker is 10%. Then a party with 1,000 CCD in the pool will receive a share of 3 CCD. If this party is a delegator, 0.3 CCD are paid as a commission to the baker and the other 2.7 CCD are kept.
+A baker opens a pool, and other CCD holders delegate some stake to this pool. The probability that the baker running the pool is chosen to bake the next block is then proportional to the total stake in the pool. Rewards are distributed to everyone in the pool proportional to their stake, and a the baker takes a commission on the earnings of the delegators. For example, suppose that a pool has 10,000 CCD. And suppose that in a 24 hour period this pool gets 30 CCD as reward and the commission of the baker is 10%. Then a party with 1,000 CCD in the pool will receive a share of 3 CCD. If this party is a delegator, 0.3 CCD are paid as a commission to the baker and the other 2.7 CCD are kept.
 
 Bounding the size of pools
 ---------------------------
-Concordium imposes two caps on the amount of stake in a single pool. The first cap bounds the size of a pool with respect to the total amount of CCD in circulation. For example, a pool cannot have more than 5% of all the CCD. This limit ensures that the pools remain distributed and prevents the crash of one baker from affecting the whole system. A party with lots of capital can run multiple bakers to avoid this cap. This would increase decentralization.
+Concordium imposes two caps on the amount of stake in a single pool. The first cap bounds the size of a pool with respect to a fraction of the total staked CCDs. For example, with the capital bound cap a pool cannot have more than 5% of all the CCD. This limit ensures that the pools remain distributed and prevents the crash of one baker from affecting the whole system. A party with lots of capital can run multiple bakers to avoid this cap. This would increase decentralization.
```

Comment:
I think it would be a good idea to either a) expand on this example and describe it in the topic, or b) expand the example and include an illustration though I don't have a clear idea of how that illustration should look at the moment. We can always add it later.